src/frontend/package.json

```diff
@@ -16,7 +16,9 @@
     "lint-all": "yarn lint-fix && yarn prettier-write",
     "build-sass": "sass scss/_main.scss ../backend/marsha/static/css/main.css --load-path=node_modules",
     "build-sass-production": "sass scss/_main.scss ../backend/marsha/static/css/main.css --style=compressed --load-path=node_modules",
-    "test": "jest"
+    "test": "jest",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
Not sure this command is relevant for our case. It just generates a static website (that does not work as it is from my side...).

Reply:
Indeed, but this command is added when we install storybook, so I leaved it.
We can remove it, though.

Reply:
Furthermore, `"build-storybook": "build-storybook"` is useless. `yarn build-storybook` is able to find this command from the `storybook` package.
I'll also update `.gitignore` accordingly.

Reply:
Do you mean add `storybook-static` to .gitignore ?